packages/formation-react/src/components/ErrorableFileInput/ErrorableFileInput.stories.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ErrorableFileInput from './ErrorableFileInput';
+
+export default {
+  title: 'Library/ErrorableFileInput',
+  component: ErrorableFileInput,
+};
+
+const Template = (args) => <ErrorableFileInput {...args} />;
+
+const defaultArgs = {
+  buttonText: 'Upload some files',
+  errorMessage: 'Error message',
```

Comment:
Could you separate the invalid vs valid states into different stories?

Reply:
my original thinking was that you could change the controls to see the different states but you're right its better to have a separate story for each state